packages/browser-tests/src/routes/info.ts

```diff
@@ -16,7 +16,7 @@ router.get("/info", (async (_req: Request, res: Response) => {
     }
 
     const result = await page.evaluate(() => {
-      return window.wakuAPI.getPeerInfo(window.waku);
+      return (window as any).wakuAPI.getPeerInfo((window as any).waku);
```

Comment:
do not do this, you can find a better place
for example `as ITestBrowser` where `ITestBrowser = typeof window & { wakuAPI }`